components/automate-cli/pkg/verifyserver/models/startMockServer.go

```diff
@@ -17,10 +17,15 @@ type Server struct {
 	Port         int
 	ListenerTCP  net.Listener
 	ListenerUDP  net.PacketConn
-	ListenerHTTP *http.Server
+	ListenerHTTP HTTPsListener
+	SignalChan   chan bool
 	Protocol     string
 }
 
 type HTTPSServerResponse struct {
 	Status string `json:"status"`
 }
+
+type HTTPsListener interface {
+	Shutdown(ctx context.Context) error
```

Comment:
everyone keeping there interface in services folder

Reply:
This interface is used in model Server on line 16